storage/google/cloud/storage/blob.py

```diff
@@ -712,7 +712,11 @@ def download_to_filename(self, filename, client=None, start=None, end=None):
             os.utime(file_obj.name, (mtime, mtime))
 
     def download_as_string(self, client=None, start=None, end=None):
-        """Download the contents of this blob as a string.
+        """Download the contents of this blob as a bytes object.
+
+        .. note::
+            the method name, ``download_as_string`` was chosen for
+            compatibility with the ``boto`` library's API.
```

Comment:
Thanks @tseaver, I'd rather not explicitly state boto in documentation and only capture "bytes" being returned. 

It does related questions about the surface design used by the Python library.